metricbeat/module/golang/heap/_meta/data.json

```diff
@@ -0,0 +1,46 @@
+{
+    "@timestamp": "2017-02-17T01:41:24.646Z",
+    "beat": {
+        "hostname": "MacBook-Pro.local",
+        "name": "MacBook-Pro.local",
+        "version": "6.0.0-alpha1"
+    },
+    "golang": {
+        "heap": {
+            "allocations": {
+                "active": 3325952,
+                "allocated": 2165152,
+                "total": 3393472,
+                "frees": 7975,
+                "idle": 1163264,
+                "mallocs": 27041,
+                "objects": 19066
+            },
+            "cmdline": "./metricbeat -httpprof=127.0.0.1:6060 -e -v",
+            "gc": {
+                "total_count": 1,
+                "total_pause_ns": 152192,
+                "next_gc_limit": 4194304,
+                "pause_ns": {
```

Comment:
`total_pause_ns` is gc pause time during lifetime of the golang program,
and the `pause_ns` is durations during the metricbeat collect period, it is a group, during the collect period,
we calculate sum, max and avg.